third-party/realdds/src/dds-stream-sensor-bridge.cpp

```diff
@@ -0,0 +1,436 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2023 Intel Corporation. All Rights Reserved.
+
+#include <realdds/dds-stream-sensor-bridge.h>
+#include <realdds/dds-device-server.h>
+#include <realdds/dds-stream-server.h>
+#include <realdds/dds-exceptions.h>
+
+#include <rsutils/easylogging/easyloggingpp.h>
+#include <rsutils/json.h>
+
+#include <algorithm>
+#include <iostream>
+
+using nlohmann::json;
+
+
+namespace realdds {
+
+
+dds_stream_sensor_bridge::dds_stream_sensor_bridge() {}
```

Comment:
Can delete empty constructor and destructor

Reply:
I'd rather keep them.